constants.js

```diff
@@ -94,7 +94,14 @@ const constants = {
         'requestPayment',
         'restore',
         'torrent',
+        'legal-hold',
```

Comment:
i know the diff will look weird, but can you put these in alphabetical order within the array? I think it'll be better for future code reading :)

Reply:
Arrays are sorted and organized now.